VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json

```diff
@@ -218,6 +218,7 @@
     "are-not-allowed-to-smoke-including-e-cigarettes": "are not allowed to smoke (including e-cigarettes)",
     "must-be-alone-unless-you-have-the-judges-permission": "must be alone unless you have the judge’s permission to have someone with you",
     "must-ask-the-judges-permission-to-move-away-from-screen": "must ask the judge's permission if you need to move away from your screen",
+    "abuse-warning": "abuse will NOT be tolerated, anyone using threatening or aggressive behaviour against our staff or judges will  be removed from the hearing. You may also be reported to and arrested by the Police; you may be sent to Prison; your case may continue and be decided in your absence.",
```

Comment:
I know this probably came straight from the ticket, but there are some typos in this:

_our staff or judges will  be_ -> there's an extra space between _will_ and _be_
_Police_ -> should be lowercase p (police)
_Prison_ -> should be lowercase p (prison)

Reply:
good spot. Fixed it